### ISSUE TITLE
Missing dependency, babel-plugin-transform-object-react-jsx

### ISSUE THREAD

**Issue by gaetan-puleo** [atomiks/tippyjs#210]
Hello,
When I import the tippy.js with the version 2.4.0 I have an error : 

```
ReferenceError: Unknown plugin "transform-react-jsx" specified in "/Users/gaetanpuleo/dev/integration-common-files/node_modules/tippy.js/.babelrc" at 0, attempted to resolve relative to "/Users/gaetanpuleo/dev/integration-common-files/node_modules/tippy.js" while parsing file: /Users/gaetanpuleo/dev/integration-common-files/node_modules/tippy.js/dist/tippy.all.js
```

I think you missed to add `babel-plugin-transform-object-react-jsx` in your package.json

**Comment by atomiks** [comment]
Yeah looks like it's missing from devDeps... for some reason not all dependencies are added the the package after installing for some reason. Maybe it was a `5.6.0` npm issue, I've upgraded now.

**Comment by atomiks** [comment]
fixed in `v2.4.1`

Added missing devDep and removed unnecessary files from package
